kepler.toml

```diff
@@ -1,6 +1,8 @@
 [global]
 # log_level = "normal"
 # port = 8000
+## allow unauthorized GET requests
+# public_get = false
```

Comment:
https://github.com/spruceid/kepler/blob/main/src/main.rs#L9 The separation of levels is with a single underscore. I think I would prefer the Config to be something like 
```
Config {
  database: Database,
  orbits: Orbits
}

Orbits {
  public: bool,
  allowlist: Option<>
}
```